admin-tools/src/webapp/vm/authz-helper/chef_permissions.vm

```diff
@@ -9,7 +9,7 @@
 	<h3>
 		$thelp.getString("per.lis.title")
 	</h3>
-		#if ($alertMessage)<div class="alertMessage">$thelp.getString("gen.alert") $validator.escapeHtml($alertMessage)</div><div class="clear"></div>#end
+		#if ($alertMessage)<div class="sak-banner-error">$thelp.getString("gen.alert") $validator.escapeHtml($alertMessage)</div>#end
```

Comment:
I wonder if we should wrap all of these introductions (e.g. "Alert: ...") in the `<strong>` tags to bold them to separate them from the message.